recipes/assimp/5.x/conanfile.py

```diff
@@ -145,6 +145,7 @@ def requirements(self):
 
         self.requires("minizip/1.2.11")
         self.requires("utfcpp/3.1.2")
+        self.requires("pugixml/1.11")
```

Comment:
It's not a dependency of assimp < 5.1.0

Reply:
Thanks for spotting that, fixed. Didn't see your comment edit at first so ended up with two commits.